src/cascade_at/inputs/uncertainty.py

```diff
@@ -164,6 +164,6 @@ def stdev_from_crosswalk_version(crosswalk_version):
 
     # Do a final check on standard error
     if ((standard_error <= 0) | standard_error.isnull()).any():
-        raise ValueError("There are still negative / 0 / or null values for standard error.")
+        LOG.error("There are unrepairable negative or null values for standard error in the bundle.")
```

Comment:
Is `dismod.db` still useful in the presence of this error? I worried that this will screw up the downstream tasks. But I believe `dismod` does its own checking and if you think `dismod.db` is valuable this is alright :)